tests/main.nf.test

```diff
@@ -73,10 +73,30 @@ nextflow_pipeline {
             assert iridanext_samples.CSE.findAll { it.path.contains("Assembly/FinalAssembly") }.size() == 0
             assert iridanext_samples.CSE.findAll { it.path.contains("Assembly/FinalReports/FlattenedReports/CSE.flat_sample.json.gz") }.size() == 0
 
-            // output metadata
-            //assert iridanext_metadata.CSE."AssemblyCompleted" == false
+            // metadata
+            assert iridanext_metadata.CSE.containsKey("QC Status")
+            assert iridanext_metadata.CSE."QC Status" == "FAILED"
+            assert !iridanext_metadata.CSE.containsKey("Checkm Status")
+            assert !iridanext_metadata.CSE.containsKey("Checkm Value")
+            assert !iridanext_metadata.CSE.containsKey("Average Coverage Status")
+            assert !iridanext_metadata.CSE.containsKey("Average Coverage Value")
+            assert !iridanext_metadata.CSE.containsKey("n50 Status")
+            assert !iridanext_metadata.CSE.containsKey("n50 Value")
+            assert !iridanext_metadata.CSE.containsKey("Raw Average Quality Status")
+            assert !iridanext_metadata.CSE.containsKey("Raw Average Quality Value")
+            assert !iridanext_metadata.CSE.containsKey("Length Status")
+            assert !iridanext_metadata.CSE.containsKey("Length Value")
+            assert !iridanext_metadata.CSE.containsKey("nr contigs Status")
+            assert !iridanext_metadata.CSE.containsKey("nr contigs Value")
+            assert iridanext_metadata.CSE."QC Summary" == "[FAILED] Sample is likely contaminated, REISOLATION AND RESEQUENCING RECOMMENDED\nPassed Tests: 0/6\nSpecies ID: null"
+
+            assert iridanext_metadata.CSE."Downsampled" == false
+            assert !iridanext_metadata.CSE.containsKey("Species")
+            assert !iridanext_metadata.CSE.containsKey("GC (%)")
+            //assert iridanext_metadata.CSE."Mean Sequence Length Forward" == 150
```

Comment:
In the situation where an assembly fails (QC Summary is *[FAILED] Sample is likely contaminated*), the Mean Sequence Length Forward is unset. Is this expected?

Reply:
I was just reviewing the file, the cause of the "Mean Sequence Length Forward" being unset is a result of the "skip" option `skip_raw_read_metrics` being set to true in line 26 ( https://github.com/phac-nml/mikrokondo/blob/457c0c825d38796f73625e785c5acb7c11c9dd9a/tests/main.nf.test#L29 ). 

I beleive we initially set this value to false due to some earlier issues with scripts in CI tasks, or just to speed up CI tests as the process can be quite slow.

Reply:
Ah, yes. Of course. We are skipping raw read metrics for tests. Thanks.